modules/reitit-middleware/src/reitit/ring/middleware/exception.clj

```diff
@@ -98,6 +99,7 @@
 (def default-handlers
   {::default default-handler
    ::ring/response http-response-handler
+   ::http-response/response http-response-handler
```

Comment:
No need to add the dependency except for tests if you just spell out this keyword in full. Besides, for the dependency to actually work, it'd need to be added to `modules/reitit-middleware/project.clj`.
```suggestion
   :ring.util.http-response/response http-response-handler
```

Reply:
Thanks, I've made changes based on your suggestion.
I've moved the dependency on `metosin/ring-http-response` to the dev profile in project.clj - hope that's the right place.

Reply:
Yeah, that's good.